inventory/models.py

```diff
@@ -77,6 +77,6 @@ class Device(models.Model):
     comment = models.TextField(null=True, blank=True)
 
     def __str__(self):
-        if self.owner:
-            return "{owner}'s {name}".format(owner=self.owner.login, name=str(self.model))
+        if self.assignee:
+            return "{assignee}'s {name}".format(assignee=self.assignee.login, name=str(self.model))
```

Comment:
Btw, I suggest using [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) instead of `format()`, as it makes the code cleaner:

```suggestion
            return f"{self.assignee.login}'s {self.model}"
```

Reply:
Will it work well with the translation?

Another reason why I normally use `.format()` is that it gives better control on the formatting and decouples the string from the context (again good for translation).

Reply:
What do you mean by translation in this case?

Reply:
For completeness, I meant the Django i18n tools, and they do not support this format, so we should not use it for any strings that are meant to be internationalised.

(I will address this particular change in a separate patch and roll back here.)